### ISSUE TITLE
Can't login on Admin Page

### ISSUE THREAD

**Issue by kayusgold** [Program-O/Program-O#428]
Program O has encountered an error. this may help:
    Error # 1024
    Message: Not sure what it is, but there's a problem with checking the current version on GitHub. Maybe this will help: "Failed to connect to raw.githubusercontent.com port 443: Connection refused"

**Comment by Dave-Morton** [comment]
That particular problem shouldn't prevent you from logging in if you're using the most recent version on the `DEV` branch, but I'll try to take a look when I'm not quite so under the weather.

**Comment by patrickschur** [comment]
@kayusgold Looks like the connection was blocked by a proxy or something like that. Can you connect to this site manually?

**Comment by OverlanAI** [comment]
Yes, try localhost if your running it on your own computer. Off hand 443 is https and should not be blocked but also apache needs to be set up for https v. hosts.
If your running it locally however none of that will matter since your behind your router.
443 should not be blocked in ether case.
tnx

**Comment by kayusgold** [comment]
Thanks so much for your swift replies. I had already solved the issue. I commented out the function that check for version. I know that this is not the best idea.

On localhost, it works well. I am having this issue and many others on a free shared host that I put it on for a test. Another problem I faced was wrong url, basename etc. They are all sorted too. So I think the issue is the hosting provider. 

Thanks once again.
